pyquil/_parser/PyQuilListener.py

```diff
@@ -130,8 +144,20 @@ def exitGate(self, ctx):
         else:
             self.result.append(Gate(gate_name, params, qubits))
 
-    def exitMeasure(self, ctx):
-        # type: (QuilParser.MeasureContext) -> None
+    def exitCircuitGate(self, ctx: QuilParser.CircuitGateContext):
+        """
+        PyQuil has no constructs yet for representing gate definitions within a DEFCIRCUIT (ie. gates where the qubits
```

Comment:
do you mean `gate definitions` --> `gate instructions`? I take it that this is about any gate instructions inside of a circuit def and not about def gates in a circuit def?